### DISCUSSION TITLE
Will xplr support for FTP/SFTP view？

### DISCUSSION THREAD

**Discussion by bioinformatist** [sayanarijit/xplr#520] [Ideas]
Excuse me...I don't know if it is polite for this feature request as xplr is a TUI program, but will xplr add FTP/SFTP support? Maybe just two panels like [WinSCP](https://winscp.net/eng/index.php):

![image](https://user-images.githubusercontent.com/17194719/197707225-fef4e2c5-2419-4ba4-b330-f13cc7555e77.png)

**Comment by sayanarijit** [comment]
Sound like a nice plugin idea.

It should be fairly easy to make a plugin or custom config that will temporarily mount scp/ftp exports, and open it in a [separate pane](https://github.com/sayanarijit/dual-pane.xplr).

Can be integrated with the dual-pane plugin to reuse code.

**Reply by bioinformatist** [reply]
Ahhh...may be I can try to bring the idea into realise. Or as far as you know, is anybody doing this now?

**Reply by sayanarijit** [reply]
I don't think anyone's doing it, so go ahead and try. Feel free to post here or in discord/matrix chat if you need help.

**Reply by dtomvan** [reply]
> Can be integrated with the dual-pane plugin
How would you do that? By calling one of the `xplr.fn.custom` functions it sets?

**Reply by sayanarijit** [reply]
> By calling one of the xplr.fn.custom functions it sets?

Correct. It's also possible to expose the function and do `require('dual-pane').exposed_function()`, but I don't think it's needed.

**Reply by erikLundstedt** [reply]
In the meantime
There is a program (in the aur) called termscp that I found while expecting to only find the wine winscp wrapper package

https://github.com/veeso/termscp

Apparently it's also written in rust

One idea is to call that for "scp operations " in some way or otherwise integrate them in some way

Instead of rewriting the wheel I mean

**Reply by erikLundstedt** [reply]
In the meantime
There is a program (in the aur) called termscp that I found while expecting to only find the wine winscp wrapper package

https://github.com/veeso/termscp

Apparently it's also written in rust

One idea is to call that for "scp operations " in some way or otherwise integrate them in some way

Instead of rewriting the wheel I mean

**Reply by bioinformatist** [reply]
Got it 😄
